databricks-sdk-java/src/main/java/com/databricks/sdk/core/commons/CommonsHttpClient.java

```diff
@@ -84,8 +84,7 @@ private Response computeResponse(Request in, CloseableHttpResponse response) thr
     boolean streamResponse =
         in.getHeaders().containsKey("Accept")
             && !APPLICATION_JSON.getMimeType().equals(in.getHeaders().get("Accept"))
-            && hs.containsKey("Content-Type")
```

Comment:
This is needed because header keys are not case-sensitive (esp. if response is HTTP 2.0, which enforces header keys to be lower case).